extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -270,6 +271,13 @@ public void onConnectionLost(HueBridge bridge) {
     public void onConnectionResumed(HueBridge bridge) {
         logger.debug("Bridge connection resumed. Updating thing status to ONLINE.");
         updateStatus(ThingStatus.ONLINE);
+        // now also re-initialize all light handlers
+        for (Thing thing : getThing().getThings()) {
+            ThingHandler handler = thing.getHandler();
+            if (handler != null) {
+                handler.initialize();
```

Comment:
> before the light handler could initialize itself. 

 I think you are talking about a different "initialize" here. The method ThingHandler.initialize() must be called by the framework only and since the hue binding is a kind of reference implementation, we should not provide bad examples, but rahter stick to this rule.
It seems to me that what you are really wanting to call is initializeThing() (btw, not a very self-describing name), but you cannot because it is private - well, this could be changed and then it can be avoided that initialize() is called by the binding itself.